controller_manager/doc/userdoc.rst

```diff
@@ -391,3 +391,51 @@ Hardware and Controller Errors
 
 If the hardware during it's ``read`` or ``write`` method returns ``return_type::ERROR``, the controller manager will stop all controllers that are using the hardware's command and state interfaces.
 Likewise, if a controller returns ``return_type::ERROR`` from its ``update`` method, the controller manager will deactivate the respective controller. In future, the controller manager will try to start any fallback controllers if available.
+
+Support for Asynchronous Updates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For some applications, it is desirable to run a controller at a lower frequency than the controller manager's update rate. For instance, if the ``update_rate`` for the controller manager is 100Hz, the sum of the execution times of all controllers' ``update()`` calls must be below 10ms. If one controller requires 15ms, it cannot be executed synchronously without affecting the overall ros_control update rate. Running a controller asynchronously can be beneficial in this scenario.
+
+The async update support is transparent to each controller implementation. A controller can be enabled for asynchronous updates by setting the ``is_async`` parameter to ``true`` and specifying an ``update_rate`` that is lower than the controller manager's rate. The controller manager will load the controller accordingly. The description of these parameters can be found in the :ref:`ros2_control_parameters` section. For example:
```

Comment:
```suggestion
The async update support is transparent to each controller implementation. A controller can be enabled for asynchronous updates by setting the ``is_async`` parameter to ``true``. The controller manager will load the controller accordingly. The description of these parameters can be found in the :ref:`ros2_control_parameters` section. For example:
```

Having an update rate lower than the controller manager rate is not a necessity. It's better not to specify that